internal/declarative/v2/kube_factory_proxy.go

```diff
@@ -114,23 +112,3 @@ func (s *SingletonClients) RESTClient() (*rest.RESTClient, error) {
 	}
 	return restClient, nil
 }
-
-// Validator returns a schema that can validate objects stored on disk.
-func (s *SingletonClients) Validator(
```

Comment:
This one implement helm [factory interface](https://github.com/helm/helm/blob/main/pkg/kube/factory.go#L50), and it's a tech debt after helm deprecation.